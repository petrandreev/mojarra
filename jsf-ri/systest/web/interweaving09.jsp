<%@page pageEncoding="UTF-8"%>
<%@taglib prefix="f" uri="http://java.sun.com/jsf/core"%>
<%@taglib prefix="h" uri="http://java.sun.com/jsf/html"%>


<f:view>
    <html>
        <body>
            <h:form id="form">
                <h:dataTable value="#{interweaving09.items}" var="item"
binding="#{interweaving09.table}">
                    <h:column>
                        <%-- 
                           Validates CASE 2 in UIComponentClassicTagBase when
                           using a component binding with a lifetime longer
                           than request
                        --%>
                        <h:outputText value="#{item}"/>                     
                        ciao
                        <h:outputText value="#{item}"/>
                    </h:column>
                </h:dataTable>                
            </h:form>
        </body>
    </html>
</f:view>

