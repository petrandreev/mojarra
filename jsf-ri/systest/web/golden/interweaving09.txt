





    <html>
        <body>
            
<form id="form" name="form" method="post" action="/jsf-systest/faces/interweaving09.jsp;jsessionid=b6c56e1112007b33b36de4d24f7d" enctype="application/x-www-form-urlencoded">
<input type="hidden" name="form" value="form" />
<input type="hidden" name="javax.faces.ViewState" id="javax.faces.ViewState" value="j_id27:j_id28" />
<table>
<tbody>
<tr>
<td>Value
                        
                                             
                        ciao
                        Value</td>
</tr>
</tbody>
</table>

</form>
        </body>
    </html>


